### PR TITLE
Reformat using 'clang-format version 7.0.0 (tags/google/stable/2018-04-24)'

### PR DESCRIPTION
**Pull request by mikelehen**
This is the version of clang-format that travis is currently using.

### CODE REVIEW

File: Example/Core/Tests/FIRComponentContainerTest.m [hunk 1]
```diff
@@ -55,7 +55,7 @@ - (void)testRegisteringNonConformingClass {
 }
 
 - (void)testComponentsPopulatedOnInit {
-  FIRComponentContainer *container = [self containerWithRegistrants:@[ [FIRTestClass class] ]];
+  FIRComponentContainer *container = [self containerWithRegistrants:@ [[FIRTestClass class]]];
```
Comment (wilhuff) [inline Example/Core/Tests/FIRComponentContainerTest.m]: This seems pretty broken. Does this still work as far as the compiler is concerned?

Maybe pull out a local reference to the class to make this format right?

However that solution stinks in `testRemoveAllCachedInstances`.

Comment (mikelehen) [reply Example/Core/Tests/FIRComponentContainerTest.m]: Hrm, yeah...  but I'm not very interested in rewriting code in order to work around current clang-format annoyances.  I think we should either live with the annoyances (they'll likely go away in a future clang-format update?) or downgrade travis.  I am fine with either, though I don't know how to do the latter.

Comment (mikelehen) [reply Example/Core/Tests/FIRComponentContainerTest.m]: (Oh, and yes, the code still compiles and passes tests)

Comment (paulb777) [reply Example/Core/Tests/FIRComponentContainerTest.m]: We can control which version of clang-format we install to the travis instance.  See https://github.com/firebase/firebase-ios-sdk/blob/master/.travis.yml#L15. 

But I'm in favor of upgrading to stay current and minimize onboarding hurdles.

**Comment by mikelehen** [comment]
Maybe.  Though as long as there's no specific version specified in our repo somehow, I think there will still be room for deviation (and there isn't a way to know exactly what style.sh should check for).  It might make more sense to just remove the checks in style.sh since travis will yell at you if you used the wrong version anyway...?

**Pull request merged**